rockcraft.yaml

```diff
@@ -1,22 +1,44 @@
+---
 name: gubernator
-version: "2.7.5"
+version: "3.0.0"
 summary: High-performance, distributed rate-limiting service
 description: |
-  Gubernator is a high-performance, distributed rate-limiting service. It can be used as a standalone service or embedded as a library within your application.
+  Gubernator is a high-performance, distributed rate-limiting service.
 license: Apache-2.0
 
-platforms:
-  amd64:
-  arm64:
+platforms: {amd64, arm64}
 
-base: bare
-build-base: ubuntu@24.04
+base: ubuntu@24.04
 
 parts:
+  go-deps:
+    plugin: nil
+    build-packages:
+      - golang-go
+
   gubernator:
     plugin: go
-    source: github.com/gubernator-io/gubernator
+    source: https://github.com/gubernator-io/gubernator.git
     source-type: git
-    source-tag: v2.7.5
+    source-tag: v3.0
     stage-packages:
       - ca-certificates
+    after:
+      - go-deps
+
+services:
+  gubernator:
+    override: replace
+    startup: enabled
+    command: /bin/gubernator
+    environment:
+      GUBER_TRACING_LEVEL: DEBUG
+      GUBER_HTTP_ADDRESS: 0.0.0.0:80
+      GUBER_GRPC_ADDRESS: 0.0.0.0:81
+
+checks:
+  online:
+    override: replace
+    exec:
+      command: /bin/healthcheck
```

Comment:
Is this something provided by Gubernator?

Reply:
Yes it is.

Reply:
I'll add a comment here.